charts/argo-cd/templates/redis/deployment.yaml

```diff
@@ -25,6 +25,9 @@ spec:
         app.kubernetes.io/component: {{ .Values.redis.name }}
     spec:
       automountServiceAccountToken: false
+      {{- if .Values.global.securityContext }}
+      securityContext: {{- toYaml .Values.global.securityContext | nindent 8 }}
+      {{- end }}
```

Comment:
Hi all,
That PR didn't add a similar entry for `templates/dex/deployment.yaml` with a `global.securityContext`. Was that simply an omission that we should fix?

Reply:
I guess it was just forgotten - does it work with a similar security-context? I can add a PR.

Reply:
Yes, I think so. I've been using the same lines in `dex/deployment.yaml` as what you had here for `redis/deployment.yaml` and it's been working fine. Thanks for the PR.

Reply:
https://github.com/argoproj/argo-helm/pull/248